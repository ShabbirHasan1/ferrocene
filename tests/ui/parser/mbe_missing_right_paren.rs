// error-pattern: this file contains an unclosed delimiter
macro_rules! abc(Ø¼

// ferrocene-annotations: fls_xa7lp0zg1ol2
// Declarative Macros
