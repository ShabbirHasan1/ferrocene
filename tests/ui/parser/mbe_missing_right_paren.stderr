error: this file contains an unclosed delimiter
  --> $DIR/mbe_missing_right_paren.rs:5:23
   |
LL | macro_rules! abc(Ø¼
   |                 - unclosed delimiter
...
LL | // Declarative Macros
   |                       ^

error: aborting due to previous error

